app/services/twilio_api_service.rb

```diff
@@ -12,6 +9,14 @@ class NotificationService
   TWILIO_TEST_SMS_NUMBER = "+15005550006"
   TWILIO_TEST_WHATSAPP_NUMBER = "+14155238886"
 
+  class Error < StandardError
```

Comment:
I'm open to leaving out this custom error class and letting sentry pick up the twilio errors.

Reply:
Nah the extra context is useful. If we can't easily attach that context to the twilio error itself, then a custom error is probably more useful 👍